docs/Releasing-the-plugin.md

```diff
@@ -0,0 +1,25 @@
+# Releasing the performance plugin
+
+This document describes the steps to release the Performance plugin.
+
+## Preparing the release
+
+> The following changes need to be made through a pull request.
+
+### Update the version number
+
+The version number needs to be updated in the following files:
+
+- package.json
+- load.php
+- readme.txt
+
+### Update readme.txt
+
+Run `npm run readme -- -m "X.Y.Z"` to update the readme.txt file with the release changelog, with X.Y.Z as the release milestone name.
+
+## Create a new GitHub release
```

Comment:
Nit-pick, but since we're saying "Preparing the release" in the h2 above, let's maybe say "Publishing the release" in the h2 here?

Reply:
Yes, that will be more coherent.